src/components/enterprise-user-subsidy/data/hooks/hooks.js

```diff
@@ -244,3 +245,28 @@ export function useCustomerAgreementData(enterpriseId) {
 
   return [customerAgreement, isLoading];
 }
+
+export function useSubsidyLearnerCredit(enterpriseId, userID) {
+  const [learnerCreditSubsidies, setLearnerCreditSubsidies] = useState();
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    fetchSubsidyLearnerCredit(enterpriseId, userID)
+      .then((response) => {
+        const results = camelCaseObject(response.data);
+        if (results.length) {
+          setLearnerCreditSubsidies(results);
+        } else {
+          setLearnerCreditSubsidies(null);
```

Comment:
nit: To be consistent with the data type for `learnerCreditSubsidies` when there is data returned, should the "empty state" value be an empty array?